products/samsungmobile.md

```diff
@@ -30,27 +135,112 @@ releases:
     support: true
     eol: false
     releaseDate: 2022-04-01
+    
+-   releaseCycle: "Galaxy F23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-E236B/INS/doc.html
+    releaseDate: 2022-03-16
+
+-   releaseCycle: "Galaxy M33"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-M336BU/INS/doc.html
+    releaseDate: 2022-04-08
+
+-   releaseCycle: "Galaxy M23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-M236B/XXV/doc.html
+    releaseDate: 2022-04-08
+
+-   releaseCycle: "Galaxy A23"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-A235M/PET/doc.html
+    releaseDate: 2022-03-25
+
+-   releaseCycle: "Galaxy A13"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-A135M/PET/doc.html
+    releaseDate: 2022-03-23
+
+-   releaseCycle: "Galaxy S22 Ultra 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S908E/VAU/doc.html
+    releaseDate: 2022-02-25
+
+-   releaseCycle: "Galaxy S22+ 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S906E/XXV/doc.html
+    releaseDate: 2022-02-25
+
+-   releaseCycle: "Galaxy S22 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-S901E/XXV/doc.html
+    releaseDate: 2022-02-25
 
+-   releaseCycle: "Galaxy Tab S8 Ultra"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-X906B/XXV/doc.html
+    releaseDate: 2022-04-30
+
+-   releaseCycle: "Galaxy Tab S8+"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-X806B/XXV/doc.html
+    releaseDate: 2022-04-14
+
+-   releaseCycle: "Galaxy Tab S8"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-X706B/SER/doc.html
+    releaseDate: 2022-03-22
+
+-   releaseCycle: "Galaxy S21 FE 5G"
+    support: true
+    eol: false # https://doc.samsungmobile.com/SM-G990B2/SER/doc.html
+    releaseDate: 2022-01-07
+
+-   releaseCycle: "Galaxy Tab A8 10.5 (2021)"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-X205/INS/doc.html
+    releaseDate: 2022-01-17
+
+-   releaseCycle: "Galaxy A03"
+    support: 2022-10-01 # Approximate from here: https://9to5google.com/2022/10/04/samsung-android-12-update-rollout/
+    eol: false # https://doc.samsungmobile.com/SM-A035G/BTU/doc.html
+    releaseDate: 2022-01-21
+    
+-   releaseCycle: "Galaxy A03 Core"
```

Comment:
This comment is just a marker to indicate I did the review of the previous models. Will continue later, do not remove.

Reply:
For the record I used https://www.gsmarena.com/ for the review. @Evernow do you think it will be a good thing to link to this site ?

Reply:
I meant using the link property to point to https://www.gsmarena.com/. It is unusual for endoflife.date but documentation on this website looks way better than Samsung's own documentation.